content/docs/command-reference/diff.md

```diff
@@ -26,9 +26,9 @@ name, Git commit hash, etc.
 It defaults to comparing the current workspace and the last commit (`HEAD`), if
 arguments `a_rev` and `b_rev` are not specified.
 
-Options `--show-json` and `--show-hash` can be used to modify format and details
-of the output produced. See the [Options](#options) and [Examples](#examples)
-sections below for more details.
+Options `--show-json`, `--show-hash` and `--hide-missing` can be used to modify
```

Comment:
```suggestion
Options `--show-json`, `--show-hash`, and `--hide-missing` can be used to modify
```

Reply:
Not sure we want to mention hide-missing here though. Maybe just

> `--show-json` and `--show-hash` can be used to modify the format of the output

Reply:
reverted these changes to just omit `--hide-missing` from this part of the `dvc diff` docs

Reply:
Thanks. We probably should say "and details" now though... I'll update it separately.